HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -41,7 +41,7 @@ def test_hvac_configurations
     args_hash = { 'hpxml_path' => File.absolute_path(@tmp_hpxml_path),
                   'skip_validation' => true }
     Dir["#{@sample_files_path}/base-hvac*.xml"].each do |hvac_hpxml|
-      next unless hvac_hpxml.include? 'ground-to-air'
```

Comment:
This line only ran gshp sizing, felt like it's probably left by debugging something.

Reply:
Good catch, yes it seems like this line should be removed.

Reply:
Opened another PR to remove this line and ran the whole test set.